frontend/app/src/sphinxUI/bounty_description.tsx

```diff
@@ -0,0 +1,281 @@
+  {
+    label: 'C++',
+    border: '1px solid rgba(175, 56, 235, 0.1)',
+    background: 'rgba(175, 56, 235, 0.1)',
+    color: '#AF38EB'
+  },
+  {
+    label: 'Java',
+    border: '1px solid rgba(235, 150, 235, 0.1)',
+    background: 'rgba(235, 150, 235, 0.1)',
+    color: '#EB96EB'
+  },
+  {
+    label: 'Rust',
+    border: '1px solid rgba(224, 81, 148, 0.1)',
+    background: 'rgba(224, 81, 148, 0.1)',
+    color: '#E05194'
+  },
+  {
+    label: 'No-code',
+    border: '1px solid rgba(255, 141, 133, 0.1)',
+    background: 'rgba(255, 141, 133, 0.1)',
+    color: '#FF8D85'
+  }
+];
+
+const BountyDescription = (props: any) => {
+  const color = colors['light'];
+  const [dataValue, setDataValue] = useState([]);
+  const [descriptionImage, setDescriptionImage] = useState('');
+  // const [descriptionLoomVideo, setDescriptionLoomVideo] = useState(props?.loomEmbedUrl);
+
+  useEffect(() => {
+    if (props.description) {
+      const found = props?.description.match(/(https?:\/\/.*\.(?:png|jpg|jpeg|gif))/);
+      setDescriptionImage(found && found.length > 0 && found[0]);
+    }
+  }, [props]);
+
+  useEffect(() => {
+    let res;
+    if (props.codingLanguage.length > 0) {
+      res = LanguageObject?.filter((value) => {
+        return props.codingLanguage?.find((val) => {
+          return val.label === value.label;
+        });
+      });
+    }
+    setDataValue(res);
+  }, [props.codingLanguage]);
+
+  return (
+    <>
+      <BountyDescriptionContainer style={{ ...props.style }}>
+        <Header>
+          <div className="NameContainer">
+            <NameTag {...props} iconSize={32} isPaid={props?.isPaid} />
+          </div>
+        </Header>
+        <Description>
+          <div
+            className="DescriptionContainer"
+            style={{
+              width: descriptionImage ? '334px' : '481px'
+            }}>
+            <EuiText
+              className="DescriptionTitle"
+              style={{
+                color: props.isPaid ? color.grayish.G50 : color.grayish.G10
+              }}>
+              {props.title}
+            </EuiText>
+          </div>
+          {descriptionImage && (
+            <div className="DescriptionImage">
+              <img src={descriptionImage} alt={''} height={'100%'} width={'100%'} />
+            </div>
+          )}
+
+          {/* {props?.loomEmbedUrl && (
+            <div
+              style={{
+                height: '64px',
+                width: '130px',
+                marginLeft: '17px',
+                marginRight: '18px',
+                borderRadius: '4px',
+                overflow: 'hidden'
+              }}>
+              <iframe
+                src={props?.loomEmbedUrl + '?autoplay=1&mute=1&loop=1&controls=0'}
+                frameBorder="0"
+                style={{
+                  width: '100%',
+                  height: '100%',
+                  borderRadius: '4px'
+                }}
+              />
+            </div>
+          )} */}
```

Comment:
We might want to add a TODO here just so we don't accidentally delete this not knowing why its commented